integration-tests/oauth2/src/test/resources/versions.properties

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright 2018 Okta, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+project.version=${project.version}
+okta.sdk.version=${okta.sdk.version}
```

Comment:
Is it common to have redundant/duplicate files like this inside of a tests directory? See -> [`versions.properties`](https://github.com/okta/okta-spring-boot/pull/44/files#diff-2b9d56a3b27fcc0f134c27cfca67b451)

Reply:
Yeah, it is a bit of a PITA, but the file is scoped to the test.  There few other ways to do this (like passing args into the test runner), but this it the most flexible approach

Reply:
Yeah - I figured it was something crazy like that. Thanks!